addons/info/src/components/stories/mock-data/widget.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
A component within mock-data? Sounds like this folder should contain only data ( js|json )
?

Reply:
mmm... why a component can't be a mock-data? `addon-info` can take a React Component as an argument to show in the description section (now it's "summary"). I just need something to pass as an example to see how it works.

Reply:
But possible I already don't use it, I need to check actually